src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -435,6 +435,7 @@ public static long parseBytes(String string) throws NumberFormatException {
             case "g": case "gb": size *= 1024L; /* no break */
             case "m": case "mb": size *= 1024L; /* no break */
             case "k": case "kb": size *= 1024L; /* no break */
+            case "%": size = size * relativeUnit * 1024L / 100; break;
```

Comment:
That should probably go before the lines with no `break`...

Reply:
Ah, that was it 🙈By the way, did you know you can cache the Maven repository to speed up Travis builds?

    cache:
      directories:
      - "$HOME/repository"